src/api/routes/core-node-rpc-proxy.ts

```diff
@@ -128,6 +134,22 @@ export const CoreNodeRpcProxyRouter: FastifyPluginAsync<
     }
   }
 
+  /// Checks if we should modify all transaction fee estimations to always use the minimum fee. This
+  /// only happens if there is no fee market i.e. if the last N block tenures have not been full. We
+  /// use a threshold of 90% to determine if a block size dimension is full.
+  async function shouldUseTransactionMinimumFee(): Promise<boolean> {
+    const tenureWindow = parseInt(process.env['STACKS_CORE_FEE_TENURE_FULLNESS_WINDOW'] ?? '5');
+    const averageCosts = await fastify.db.getTenureAverageExecutionCosts(tenureWindow);
+    if (!averageCosts) return false;
+    return (
+      averageCosts.read_count < BLOCK_LIMIT_READ_COUNT * 0.9 &&
+      averageCosts.read_length < BLOCK_LIMIT_READ_LENGTH * 0.9 &&
```

Comment:
magic numbers. Could we get this from the environment variables? Reason is, unless we are sure that the values can never be changed, it is easier for anyone to adjust .env (not just the developer)